module/VuFind/src/VuFind/RecordDriver/SolrOverdrive.php

```diff
@@ -0,0 +1,662 @@
+<?php
+/**
+ * VuFind Record Driver for SolrOverdrive Records
+ *
+ * PHP version 5
+ *
+ * Copyright (C) Villanova University 2010.
+ * Copyright (C) The National Library of Finland 2015.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301
+ * USA
+ *
+ * @category VuFind
+ * @package  RecordDrivers
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Brent Palmer <brent-palmer@icpl.org>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public
+ *           License
+ * @link     https://vufind.org/wiki/development:plugins:record_drivers Wiki
+ */
+namespace VuFind\RecordDriver;
+
+use Zend\Log\LoggerAwareInterface;
+
+/**
+ * VuFind Record Driver for SolrOverdrive Records
+ *
+ * @category VuFind
+ * @package  RecordDrivers
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Brent Palmer <brent-palmer@icpl.org>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public
+ *           License
+ * @link     https://vufind.org/wiki/development:plugins:record_drivers Wiki
+ */
+class SolrOverdrive extends SolrMarc implements LoggerAwareInterface
+{
+    use \VuFind\Log\LoggerAwareTrait {
+        logError as error;
+    }
+
+    /**
+     * Overdrive Connector
+     *
+     * @var \VuFind\DigitalContent\OverdriveConnector
```

Comment:
You should add a `use` statement for this class at the top of the file instead of using the fully qualified name here, since you're using the short version in the constructor parameter list already. That would also allow you to add a typehint on the appropriate constructor parameter, which is not a bad idea.

Reply:
I think I did what you were intending here.  I also checked a few other classes for similar issues.